lib/sneeze-board.web-view.scss

```diff
@@ -23,1394 +23,3 @@ button.sneezed {
 div.names > div {
   width: 200px;
 }
-
```

Comment:
This is just a question to make sure I understand. Did some recent changes make a way for the combo box styles to work without this hack?